plugins/modules/net_tools/ldap/ldap_attrs.py

```diff
@@ -264,7 +265,8 @@ def exact(self):
     def _is_value_present(self, name, value):
         """ True if the target attribute has the given value. """
         try:
-            filterstr = "(%s=%s)" % (name, value.decode())
+            escaped_value = ldap.filter.escape_filter_chars(value.decode('utf-8'))
```

Comment:
Add `to_text` to the imports next to `to_bytes`, and use
```suggestion
            escaped_value = ldap.filter.escape_filter_chars(to_text(value))
```
instead.